openbas-front/src/admin/components/atomic_testings/atomic_testing/TargetResultsDetail.tsx

```diff
@@ -623,26 +623,26 @@ const TargetResultsDetailFlow: FunctionComponent<Props> = ({
                         <GridLegacy item={true} xs={5} sx={{ textAlign: 'end' }}>
                           {
                             injectExpectation.inject_expectation_status === 'SUCCESS' && injectExpectation.inject_expectation_type === 'PREVENTION' && (
-                              <ItemResult label="Prevented" status="Prevented" />
+                              <ItemResult label={t('Prevented')} status="Prevented" />
                             )
                           }
                           {
                             injectExpectation.inject_expectation_status === 'SUCCESS' && injectExpectation.inject_expectation_type === 'DETECTION' && (
-                              <ItemResult label="Detected" status="Detected" />
+                              <ItemResult label={t('Detected')} status="Detected" />
                             )
                           }
                           {
                             injectExpectation.inject_expectation_status === 'FAILED' && injectExpectation.inject_expectation_type === 'PREVENTION' && (
-                              <ItemResult label="Not Prevented" status="Not Prevented" />
+                              <ItemResult label={t('Not Prevented')} status="Not Prevented" />
                             )
                           }
                           {
                             injectExpectation.inject_expectation_status === 'FAILED' && injectExpectation.inject_expectation_type === 'DETECTION' && (
-                              <ItemResult label="Not Detected" status="Not Detected" />
+                              <ItemResult label={t('Not Detected')} status="Not Detected" />
                             )
                           }
                           {injectExpectation.inject_expectation_status && HUMAN_EXPECTATION.includes(injectExpectation.inject_expectation_type) && (
-                            <ItemResult label={injectExpectation.inject_expectation_status} status={injectExpectation.inject_expectation_status} />
+                            <ItemResult label={t(injectExpectation.inject_expectation_status)} status={injectExpectation.inject_expectation_status} />
```

Comment:
Translating stuff that comes from the backend is super cool, we definitely need more of that !! However might not we encountered missing translations this way if we add new statuses in the backend for instance ? Missing translation that Johanah's work on translation would not detect as missing?

Maybe we should add a comment to the list of constants in your backend class to add or update frontend translation files when changing/adding/removing values ?